---
title: Soil Microbial Functional Succession Over One Year of Human Decomposition
crossref:
  custom:
    - kind: float
      reference-prefix: Supplementary Material
      key: sup
      latex-env: supplemental
format:
  nature-pdf:
    # journal.cite-style is included in the tex file but ignored by pandoc if 
    # cite-method is not `natbib`.
    journal:
      cite-style: sn-nature
    # `citeproc` is the pandoc default. Set `cite-method: natbib` if required
    # to use the bst styles from the upstream template.
    cite-method: natbib
    keep-tex: true
    equal-margins: true
    classoption:
      lineno,
      referee
author:
  - name: Allison R. Mason
    affiliations:
      - name: University of Tennessee-Knoxville
        id: 1
        department: Department of Microbiology
        address: 1311 Cumberland Avenue
        city: Knoxville
        state: TN
        postal-code: 37996
  - name: Lois S. Taylor
    affiliations:
      - name: University of Tennessee-Knoxville
        id: 2
        department: Department of Biosystems Engineering and Soil Science
        address: 2506 E.J. Chapman Drive
        city: Knoxville
        state: TN
        postal-code: 37996
  - name: Naomi Gilbert
    affiliations:
      - ref: 1
  - name: Steven W. Wilhelm
    affiliations:
      - ref: 1
  - name: Jennifer M. DeBruyn
    affiliations:
      - ref: 1
      - ref: 2
    email: jdebruyn@utk.edu
    attributes:
      corresponding: true
abstract: |
  **Background** The succession of microbial communities during vertebrate decomposition has been observed in various settings, documenting changes in taxa as decomposition progresses. These studies have predominantly employed phylogenetic markers (*i.e.*, rRNA genes), describing community composition and structure, but ultimately are not informative of which members are active or metabolic pathways they might be expressing. This has left a foundational knowledge gap regarding the functional roles of microorganisms in vertebrate decomposition, which ultimately impact ecosystem functioning. Here we present the first known study investigating gene expression in soil impacted by human decomposition. Total RNA was extracted and metatranscriptomes obtained from soil samples collected over the course of one year from below three decomposing human bodies.
  
  **Results** Microbial gene expression profiles shifted in response to decomposition:  decomposition impacted soils were most different from controls (*i.e*. nearby soils unimpacted by decomposition) at day 86, and profiles remained altered even after one year. Shifts in gene expression were partially explained by environmental and soil physiochemical variables, including internal body accumulated degree hours (*p* = 0.001), as well as soil temperature (*p* = 0.045), pH (*p* = 0.042) and electrical conductivity (*p* = 0.037). Differential expression analysis revealed that microbes in decomposition soils displayed increased expression of stress response genes (mean fold change 3.48), particularly heat shock proteins (*p* < 0.001), whose expression increased between days 0 and 58 and remained elevated through day 376. Further, we identified genes whose expression was altered at certain timepoints. This included increased expression of nitrogen cycling genes HAO (85x), norB (83x), and nosZ (19x) at day 86 when dissolved oxygen was ~85%, suggesting that microbial communities may be converting hydroxylamine to nitric oxide during reduced oxygen conditions. 
  
  **Conclusions** Our results show that human decomposition alters soil microbial gene expression profiles providing evidence of altered microbial metabolisms (*e.g.*, taurine metabolism, nitrogen cycling, and lipid metabolism) and reveal the potential of vertebrate decomposition to have both ephemeral and lasting effects on ecosystem processing in response to mortality events. 
keywords: [Human Decomposition, Microbial Succession, Metatranscriptomics, Soil Microbial Ecology]
bibliography: bibliography.bib  
---

```{r}
#| include: false

library(here)
library(readxl)
library(tidyverse)
library(pdftools)
```

# Introduction

Soil microbial communities are important drivers of ecosystem processes in terrestrial environments. Many soil microbes are decomposers that are involved in degradation of complex organic matter and drive nutrient cycling in terrestrial ecosystems. Environmental disturbances can impact the presence and/or activity of soil microorganisms that are invovled in these cycles, ultimately affecting nutrient availability and the release of greenhouse gas emissions, such as CO\textsubscript{2} and N\textsubscript{2}O [@benninger_biochemical_2008;@towne_prairie_2000]. Vertebrate death and subsequent carcass deposition in terrestrial ecosystems is one disturbance resulting in the deposition of large quantities of organic C and N [@parmenter_carrion_2009;@macdonald_carrion_2014;@bump_ungulate_2009;@aitkenhead-peterson_mapping_2012;@keenan_mortality_2018;@fancher_evaluation_2017;@quaggiotto_dynamic_2019], along with other elements (P, K, S, *etc*) [@taylor_soil_2023], which collectively contribute to microbially-mediated biogeochemical cycling. In additon to this, changes in pH, temperature, and fluctuations in soil oxygen provide abiotic filtering further impacting microbial metabolic strategies [@aitkenhead-peterson_mapping_2012;@keenan_mortality_2018;@fancher_evaluation_2017;@taylor_soil_2023;@mason_body_2022].


While C and N transformations have been documented during decomposition, the functional response of microbes and their roles in nutrient cycles remain unclear. Previous work from our lab [@cobaugh_functional_2015;@keenan_spatial_2018] and others [@singh_temporal_2018;@metcalf_microbial_2016;@finley_microbial_2016;@fu_fungal_2019] have conducted surveys of decomposition-impacted soil microbial communities through amplicon sequencing of marker genes (*i.e.*, 16S rRNA, 18S rRNA, ITS). This has allowed us to investigate changes in microbial biodiversity and composition in response to vertebrate decomposition, revealing patterns such as increases in the anaerobic taxa *Firmicutes* and *Bacteroidetes*. However, few studies have investigated soil biogeochemistry and microbial communities within the same study, which can further help to describe microbial ecology in human and animal decomposition systems. Taylor et al. (2024) [@taylor_soil_2024] suggested that observed fungal community shifts were linked to soil dissolved oxygen, highlighting interactions between soil microbes and the surrounding environment. Additionally, Metcalf et al. (2016) [@metcalf_microbial_2016] started to connect microbial changes to physiochemical data (*e.g.*, nitrate, pH) during controlled mouse decomposition experiments. While insightful for making potential connections between taxa and physiochemistry, these analyses cannot inform which taxa are active members of the community responsible for chemical transformations, which functional pathways/genes are expressed, and how these pathways are altered in response to decomposition.

Methods such as RNA sequencing (*i.e.*, metatranscriptomics) and metabolomics can be used to investigate microbial community functional succession in response to decomposition by measuring gene expression and metabolites, respectively. Where DNA-based methods are limited to explaining structural succession, analysis of microbial community gene expression (*i.e.*, mRNA) helps to determine which functional pathways are altered in response to decomposition. This can inform how ecological functions, including C and N cycling, are impacted by decomposition events in terrestrial ecosystems. To date, only two studies have applied metatranscriptomic approaches to assess mRNA in vertebrate decomposition samples [@burcham_total_2019;@ashe_characterization_2021]: Burcham et al. (2019) [@burcham_total_2019] examined gene expression of internal organ microbial communities during mouse decomposition, while Ashe et al. (2021) [@ashe_characterization_2021] examined gene expression of oral microbial communities during human decomposition. Both studies suggest that the host microbial community functionality is altered during decomposition, including differential expression of amino acid and carbohydrate metabolism in the heart [@burcham_total_2019] and shifts in gene transcripts across different taxa [@ashe_characterization_2021]. We expect that soil microbial community gene expression profiles are also altered; however, this has never been examined to our knowledge. The decomposition-impacted soil metabolome was assessed by DeBruyn et al. (2021) [@debruyn_comparative_2021], showing changes in soil metabolites over time, however it is unclear which microbes are responsible for these shifts. Additionally, DeBruyn et al. (2021) [@debruyn_comparative_2021] showed the soil metabolome was still altered compared to starting conditions at the end of the 21-week study, suggesting long-term impacts of decomposition on soil microbial functioning.

The purpose of this study was to investigate soil microbial gene expression during a one-year period of human decomposition and address the following questions: (1) which genes are differentially expressed in soils impacted by human decomposition? (2) how does gene expression change over time in decomposition-impacted soils? (3) do microbial gene expression profiles return to pre-decomposition conditions after one year? The human body is comprised of carbon-rich organic molecules, many of which are broken down during decomposition. We hypothesized that gene expression would change over time as resources are used and transformed and soil chemical and physical conditions change as a result of tissue degradation [@keenan_mortality_2018;@fancher_evaluation_2017;@debruyn_comparative_2021]. For example, we expected to observe changes in the expression of genes encoding enzymes involved in nitrogen cycling, as increased nitrogen transformations have been previously described in decomposition soils [@keenan_mortality_2018]. Of the main macromolecules in the body (carbohydrates, proteins, lipids, and nucleic acids), we were particularly interested in lipid metabolism, as we expect lipids from the body to enter the soil during decomposition and previous studies showed an increase of lipolytic organisms in decomposition soils [@howard_characterization_2010]. Finally, multiple studies have shown than soil chemistry [@macdonald_carrion_2014;@keenan_mortality_2018] and microbial community composition [@cobaugh_functional_2015;@singh_temporal_2018] (via 16S rRNA gene amplicon sequencing) are still impacted after one year, therefore we did not expect soil expression profiles to return to pre-decomposition conditions.

To answer these questions, metatranscriptomes of soil samples collected at six key timepoints over one year of human decomposition were used to determine the active populations and expression of genes and pathways relevant to the enhanced biogeochemical cycling observed in decomposition hotspots. We compared gene expression between decomposition timepoints and control soils that were unexposed to decomposition products to identify functions or functional pathways of interest. This assessment of functional profiles within decomposition-impacted soils provided insight into the microbial response to vertebrate decomposition in terrestrial settings and biogeochemical cycling within these hotspots.

# Results

## Soil Physiochemistry

```{r}
#| include: false

do.12 <- read_excel(here("data/SP_metadata.xlsx"), 
    sheet = "R") %>%
    mutate(Trt = as.factor(Trt),
           Study_day = as.factor(Study_day),
           Stage = factor(Stage, 
                          ordered = TRUE,
                          levels = c("Initial", "Active", "Advanced4","Advanced6","Advanced11","Skeletonization4"))) %>%
    pivot_longer(c(-Sample_name, -Date, -MetaT, -Trt, -Rep, -Study_con, -Study_day, -ADH_ambient, -ADH_internal, -ADH_soil, -Stage, -TBS, -timepoint), names_to = "parameter") %>%
    filter(parameter == "DO",
           Trt == "Decomposition",
           Study_day == 12) 
    
do.12 %>%
  summarise(avg = mean(value))


nh4_mag <- read_excel(here("data/SP_metadata.xlsx"), 
    sheet = "R") %>%
    mutate(Trt = as.factor(Trt),
           Study_day = as.factor(Study_day),
           Stage = factor(Stage, 
                          ordered = TRUE,
                          levels = c("Initial", "Active", "Advanced4","Advanced6","Advanced11","Skeletonization4"))) %>%
    pivot_longer(c(-Sample_name, -Date, -MetaT, -Trt, -Rep, -Study_con, -Study_day, -ADH_ambient, -ADH_internal, -ADH_soil, -Stage, -TBS, -timepoint), names_to = "parameter") %>%
    filter(parameter == "NH4_mg_gdw",
           Trt == "Decomposition",
           Study_day %in% c(0, 12, 58)) %>%
    summarise(avg = mean(value), .by = c("Study_day")) %>%
    pivot_wider(names_from = Study_day, values_from = avg) %>%
    mutate(diff = `12`/`0`) %>%
    pull(diff)
```

Soil chemistry was altered in response to human decomposition, with multiple parameters still impacted after one year [@taylor_soil_2024]. Generally, soil pH decreased and remained low in decomposition soils of all but one individual. Soil electrical conductivity (EC) increased in response to decomposition, remaining elevated through approximately day 58 before gradually decreasing throughout the remainder of the study (@sup-chem-plot). Respiration (evolved CO~2~) increased by an order of magnitude beginning at day 12, which corresponded to a reduction in soil dissolved oxygen (DO) to `r min(do.12$value)`\% - `r max(do.12$value)`\%. Ammonium concentrations increased `r round(nh4_mag, 0)`-fold, reaching maximum concentrations between days 12 and 58. This was followed by decreased ammonium and increased nitrate concentrations at day 86, with nitrate concentrations reaching a maximum at day 168 (@sup-chem-plot).

## Sequencing

Illumina sequencing of the 24 libraries yielded a total of 5,073,476,730 reads, or 2,536,738,365 paired reads, with a mean of 105,697,432 paired reads per sample. Removal of adapters and low-quality reads removed 4.7\% of all reads, leaving 4,834,123,062 total reads. Filtering of ribosomal RNA further removed 7.3\% of reads, leaving 4,479,804,360 reads for assembly. After co-assembly, a total of 6,257,674 proteins were identified by Prodigal. From this, 1,048,573 proteins were annotated by eggNOG-mapper (16.7\%). Most of the annotated proteins were taxonomically annotated as bacteria (91.3\%), followed by eukaryotes (7.6 %), and archaea (0.81 %). Of the 7.6\% of eukaryotic proteins, 64.4\% (4.9\% of all proteins) were annotated as fungi. For this study, genes of interest included all bacterial, archaeal, and fungal proteins, therefore all non-fungal eukaryotic proteins (32,004) were removed prior to downstream analysis. The reference file of genes was then used to determine gene transcript counts in all samples using CLC genomic workbench. The percent of reads mapped to genes of interest ranged from 21\% to 38\%, with an average of 31\% reads mapped. Gene counts were then combined in a single file and used for downstream analyses in R.

## Microbial gene expression in response to human decomposition

Gene expression profiles in decomposition-impacted soils shifted away from controls and day zero samples as decomposition progressed (`Fig~\ref{fig-mds}`{=latex}A). Expression was most different from controls on study days 58, 86, 168 (@sup-heatmap-plot), before shifting back toward control conditions on study day 376. After one year of decomposition, soil gene expression profiles had not returned to pre-decomposition conditions, as evidenced by their clustering away from controls and day zero samples in the MDS plot (`Fig~\ref{fig-mds}`{=latex}A).

```{=tex}
% Place figure captions after the first paragraph in which they are cited.
\begin{figure}[!h]
\caption{{\bf Microbial gene expression profiles are altered during human decomposition.}
Multidimensional scaling (MDS) shows gene expression within soils changed as decomposition progressed (A). Additionally, canonical correspondence analysis (CCA) shows that environmental variables explained 64.3\% of the variation in gene expression profiles (B). Variables in bold red type significantly (p < 0.05) explained some of the variation in gene expression profiles as assessed by Permutational Analysis of Variance (PERMANOVA). In both panels soils from controls (CON) and the three donors (SP1, SP2, SP3) are denoted by symbol shape, while color represents study day. In B, soil physiochemical variable loadings are reporesented by arrows: Gravimetric water content (GWC), electrical conductivity (EC), pH (pH), respiration(evolved CO\textsubscript{2}\ \textmu mol gdw\textsuperscript{-1}), ammonium (NH\textsubscript{4}), and nitrate (NO\textsubscript{3}) concentrations (mg gdw\textsuperscript{-1}), percent carbon (\%C), percent nitrogen (\%N), carbon:nitrogen ratio (C:N), ambient temperature (T\textsubscript{A}), and soil temperature (T\textsubscript{S}).}
\label{fig-mds}
\end{figure}
```
![](../../figures/Fig1.png)

Some correlations were observed between gene expression shifts and soil physiochemical data at decomposition timepoints. Canonical correspondence analysis (CCA) was used to constrain gene expression data with soil physiochemical data (`Fig~\ref{fig-mds}`{=latex}B). CCA1 and CCA2 explained 36.2\% and 20.9\% of the variance in gene expression, respectively. Transcript profiles at day 12 were associated with an increase in soil carbon to nitrogen ratio (C:N). Gene expression profiles at days 58 to 86 were positively correlated with increased soil temperature, EC, and evolved CO~2~, while study day 168 was associated with elevated levels of soil NO~3~. Further, Permutational Analysis of Variance (PERMANOVA) revealed that internal accumulated degree hours (ADH), soil temperature, pH, and EC significantly explained some of the variation in gene expression profiles (p < 0.05). No other soil chemical variables were significant at \textalpha\ = 0.05 (@sup-tbl-cca-panova).

```{r}
#| include: false

top40_trt <- read_csv(here::here("data/top_40_DE_trt.csv"))

cog1 <- "Post-translational modification, protein turnover, and chaperones"

downreg_dist <- top40_trt %>%
  filter(trt_cont == -1) %>%
  count(Ann) %>%
  mutate(per = n / sum(n) * 100)
  
ppc_per <- downreg_dist %>%
  filter(Ann == cog1) %>%
  pull(per)

```

Overall, decomposition changed soil gene expression profiles over the one-year study relative to control soils. Differential expression analysis between decomposition and control soils identified 7,047 down-regulated and 38,425 up-regulated genes. Gene transcripts that were associated with control soils comprised a wide variety of clusters of orthologous genes (COG) functional categories. Specifically, the top 20 genes whose expression was higher in control soils belonged to ten unique COG categories, including signal transduction mechanisms, transcription, and those of unknown function. In contrast, the top 20 genes whose expression was higher in decomposition soils only fell into four COG categories (@sup-t30-trt-plot A): 1) post-translational modification, protein turnover, and chaperones; 2) energy production and conversion; 3) cell motility; and 4) carbohydrate transport and metabolism. The most common COG category represented in decomposition soils (`r ppc_per`\%) were post-translational modification, protein turnover, and chaperones. Within this category, several heat shock stress response genes were identified, including SSA2, HSP82, and clpB (@sup-t30-trt-tbl). Further investigation into these genes shows their expression increased in response to decomposition, typically reaching maximum transcript levels around study days 58 and 86 (`Fig~\ref{fig-heatshock}`{=latex}). This corresponded to elevated soil temperatures below decomposing bodies between study days 12-80, with soil temperatures increasing to approximately 43&deg;C [@taylor_soil_2024], and maximum soil EC measurements between days 12 and 58 (@sup-chem-plot).

```{=tex}
% Place figure captions after the first paragraph in which they are cited.
\begin{figure}[!h]
\caption{{\bf Normalized log2 expression of heat shock proteins identified by differential expression analysis comparing decomposition and control soils.}
Each panel represents a single heat shock transcript, labled with query ID. Symbol color denotes if the sample is a control (CON, green), or one of three individuals: SP1 (orange), SP2 (purple), or SP3 (pink).}
\label{fig-heatshock}
\end{figure}
```
![](../../figures/Trt_DE_top15_heatshockTPM.png)

Taxonomy associated with top differentially expressed gene transcripts also differed between control and decomposition soils. The top 40 significantly differentially expressed gene transcripts in decomposition soils were associated with Fungi, *Actinobacteria*, and *Xanthomonadales*, while gene transcripts in controls were associated with *Acidobacteria*, *Cyanobacteria*, *Proteobacteria* (\textalpha, \textdelta, \textgamma), and Planctomycetes (@sup-t30-trt-plot B). The greatest number of differentially expressed genes relative to control samples was observed at day 86, where we saw 145,460 and 124,883 up- and down-regulated genes, respectively.

## Fate of decomposition products as evidenced in gene expression profiles over time

Differential expression analysis between respective sequential study days further revealed which genes were altered between decomposition timepoints.  The top ten significantly up- and down-regulated genes, determined by the lowest p-values from differential expression analysis (\alpha < 0.05), are reported in @sup-t20-day-tbl and `Fig~\ref{fig-top-day-bar}`{=latex}. 

```{=tex}
% Place figure captions after the first paragraph in which they are cited.
\begin{figure}[!h]
\caption{{\bf Top twenty up- and down-regulated genes in decomposition soils comparing sequential study days (0, 12, 58, 86, 168, 376) colored by COG functional category (A) and taxonomic annotation (B).}
Positive values denote increased expression compared to the preceding timepoint, while negative values denote a decrease.}
\label{fig-top-day-bar}
\end{figure}
```
![](../../figures/Study.con_DE_top20_barplot.png)

Expression of genes annotated with the COG categories cell wall/membrane/envelope biogenesis, inorganic ion transport and metabolism, and carbohydrate transport and metabolism increased from day 0 to 12. In contrast, expression of secondary metabolite biosynthesis, transport, and catabolism genes decreased during this period (`Fig~\ref{fig-top-day-bar}`{=latex}A). Transcripts from *Bacilli* and *Clostridia* increased, while transcripts from *Actinobacteria* decreased between study days zero and 12 (`Fig~\ref{fig-top-day-bar}`{=latex}). 

Between days 12 and 58, 90\% of the topmost upregulated genes were associated with the COG translation, ribosomal structure and biogenesis and all were taxonomically associated with *Betaproteobacteria* (`Fig~\ref{fig-top-day-bar}`{=latex}A,B). Many of these genes were annotated as proteins within the rpl protein family, involved in ribosomal binding. Genes across multiple COG categories with taxonomic associations to *Bacilli* and *Clostridia* decreased between study days 12 and 58, six of which were transcripts that previously increased between days zero and 12 (`Fig~\ref{fig-top-day-bar}`{=latex}B, @sup-t20-day-tbl). 

```{r}
#| include: false

hao_aov <- read.csv(here("data/hao_lmer_anova.csv"))
```

Multiple transcripts associated with the energy production and conversion COG, as well as transcripts annotated with the COGs inorganic transport and metabolism, and translation, ribosomal structure and biogenesis, increased between days 58 and 86 (`Fig~\ref{fig-top-day-bar}`{=latex}A). Two of the upregulated energy and production and conservation transcripts were associated with cytochrome c oxidase subunits in *Betaproteobacteria*, while another was annotated as *hao*, encoding the enzyme hydroxylamine dehydrogenase which is involved in conversion of hydroxylamine to nitrite during nitrification (@sup-t20-day-tbl). Further investigation into hydroxylamine dehydrogenase showed a significant increase in *hao* transcripts at day 86 followed by subsequent decreases at days 168 and 376 (F = `r round(hao_aov$F.value, 3)`; *p* = `r round(hao_aov$Pr..F., 3)`). This increase corresponded to decreased soil ammonium levels and subsequent accumulation of nitrate (@sup-chem-plot). Half of the topmost downregulated genes between days 58 and 86 were not assigned to a COG (*i.e.*, unclassified) or were of unknown function. 

Differential expression comparing study days 86 with 168 and 168 with 376 identified genes across a variety of functional categories, with many unclassified in the COG database or with unknown function (`Fig~\ref{fig-top-day-bar}`{=latex}A). Expression of carbohydrate transport and metabolism genes associated with *Bacilli* decreased between day 168 and 376. Additionally, *Acidobacteria* transcripts increased in decomposition-impacted soils between study day 168 and 376 (`Fig~\ref{fig-top-day-bar}`{=latex}B). These transcripts were not associated with any single COG category, however.

## Carbon-dominant molecules

```{r}
#| include: false

lipase_full_aov <- read_csv(here("data/lipase_TPM_full_anova.csv"))
full_sd_f <- round(lipase_full_aov[lipase_full_aov$`...1` == "Study_day", "F value"], 3)
full_sd_p <- if_else(lipase_full_aov[lipase_full_aov$`...1` == "Study_day", "Pr(>F)"] < 0.001, "< 0.001", as.character(round(lipase_full_aov[lipase_full_aov$`...1` == "Study_day", "Pr(>F)"], 3)))

full_i_f <- round(lipase_full_aov[lipase_full_aov$`...1` == "Study_day:tax", "F value"], 3)
full_i_p <- if_else(lipase_full_aov[lipase_full_aov$`...1` == "Study_day:tax", "Pr(>F)"] < 0.001, "< 0.001", as.character(round(lipase_full_aov[lipase_full_aov$`...1` == "Study_day:tax", "Pr(>F)"], 3)))

lipase_bact_aov <- read_csv(here("data/lipase_TPM_bact_anova.csv"))
bact_f <- round(lipase_bact_aov[lipase_bact_aov$`...1` == "Study_day", "F value"] , 3)
bact_p <- if_else(lipase_bact_aov[lipase_bact_aov$`...1` == "Study_day", "Pr(>F)"] < 0.001, "< 0.001", as.character(round(lipase_bact_aov[lipase_bact_aov$`...1` == "Study_day", "Pr(>F)"], 3)))

lipase_fung_aov <- read_csv(here("data/lipase_TPM_fungi_anova.csv"))
fung_f <- round(lipase_fung_aov[lipase_fung_aov$`...1` == "Study_day", "F value"], 3)
fung_p <- if_else(lipase_fung_aov[lipase_fung_aov$`...1` == "Study_day", "Pr(>F)"] < 0.001, "< 0.001", as.character(round(lipase_fung_aov[lipase_fung_aov$`...1` == "Study_day", "Pr(>F)"], 3)))
```

We expected to observe increased expression of lipid metabolizing genes during active and advanced decomposition as microbes degraded lipids deposited in the soil [@howard_characterization_2010]. Therefore, we investigated changes in triacylglycerol lipase (enzyme commission number: 3.1.1.3) gene transcription in our soils. Generally, lipase transcripts decreased as decomposition progressed (HLM F = `r full_sd_f`, *p* `r full_sd_p`), however we also observed a significant interaction between study day and taxonomic annotation (F = `r full_i_f`; *p* `r full_i_p`). Specifically, lipase gene transcripts annotated as bacteria decreased with decomposition time (F = `r bact_f`; *p* = `r bact_p`), while fungal lipase transcripts increased, reaching a maximum at study day 58 (F = `r fung_f`; *p* = `r fung_p`) (`Fig~\ref{fig-lipase}`{=latex}). 

```{=tex}
% Place figure captions after the first paragraph in which they are cited.
\begin{figure}[!h]
\caption{{\bf Mean transcript abundance, in transcripts per million (TPM), of all bacterial (A) and fungal (B) triacylglycerol lipase (EC 3.1.1.3) genes over time.}
Black lines (A, B) report mean and standard deviation of TPM from three individuals (black line), while gold stars denote mean TPM in control soils. P-values are the result of ANOVAs where average TPM and study day are the dependent and independent variables, respectively, while letters are the result of post-hoc Tukey tests between decomposition timepoints. In B, bars show the relative abundance of the fungal classes \textit{Saccharomycetes}, \textit{Sordariomycetes}, and \textit{Eurotiomycetes}, reported in Taylor et al. (2024).}
\label{fig-lipase}
\end{figure}
```
![](../../figures/Fig4.png)

## Nitrogen- and sulfur-enriched molecules

Expression of nitrogen cycling genes was impacted in response to human decomposition. Due to the detection of *hao* in our differential expression analysis, and our hypotheses predicting changes to nitrogen transformation processes, the expression of genes encoding common enzymes involved in nitrogen cycling (*nifH*, *nirB*, *nirK*, *norB*, *nosZ*, *nrfA*, *nxrA*, and *amoA*) were assessed using their enzyme commission numbers (`Fig~\ref{fig-n-genes}`{=latex}A,B). *nifH*, encoding a subunit of nitrogenase which is involved in nitrogen fixation, displayed little to no changes in gene expression between control and decomposition soils. Transcripts for two genes encoding enzymes contributing to the last two steps of denitrification, *norB* (encodes nitric oxide reductase) and *nosZ* (encodes nitrous oxide reductase), increased between study days 12 and 86, and decreased at study day 168 before increasing again at day 376. In contrast, expression of genes encoding nitrate reductase, *narG*, and NO-forming nitrite reductase, *nirK*, remained low until day 376 when transcripts for both genes increased. As noted above, expression of *hao*, encoding hydroxylamine dehydrogenase, increased at study day 86 before decreasing at remaining timepoints (`Fig~\ref{fig-top-day-bar}`{=latex}A, `Fig~\ref{fig-n-genes}`{=latex}B). Expression of *amoA*, encoding a subunit of ammonia monooxygenase, and *nxrA*, encoding a subunit of nitrite oxidoreductase, which are involved in nitrification, changed in response to decomposition. *amoA* transcripts initially decreased at day 12, remaining reduced until study day 376. Similarly, abundance of genes that encode for enzymes involved in dissimilatory nitrate reduction, *nirB*, and *nrfA*, was low for the first 168 days, with *nrfA* expression increasing at day 376 (`Fig~\ref{fig-n-genes}`{=latex}B). 

```{=tex}
% Place figure captions after the first paragraph in which they are cited.
\begin{figure}[!h]
\caption{{\bf Average gene expression, in transcripts per million (TPM), of commonly used marker genes for enzymes involved in nitrogen cycling over time in controls (A) and decomposition (B) soils.}
Data in B represent mean and standard deviation of TPM from three individuals.}
\label{fig-n-genes}
\end{figure}
```
![](../../figures/Fig5.png)

```{r}
#| include: false

taud_lmer <- read_csv(here("data/taud_aov.csv"))
taud_f <- round(taud_lmer[1, "F value"], 3)
taud_p <- round(taud_lmer[1, "Pr(>F)"], 3)

bsh_lmer <- read_csv(here("data/bsh_aov.csv"))
bsh_f <- round(bsh_lmer[1, "F value"], 3)
bsh_p <- if_else(bsh_lmer[1, "Pr(>F)"] < 0.001, "< 0.001", as.character(round(bsh_lmer[1, "Pr(>F)"], 2)))
```

Expression of genes involved in metabolism of nitrogen and sulfur-containing compounds were also impacted by human decomposition. Specifically, four of the top ten genes whose expression decreased at day 12 were related to taurine metabolism, with their annotations associated with *tauD*, encoding taurine dioxygenase. (@sup-t20-day-tbl). Further investigation into *tauD* showed that mean expression of these genes decreased steadily over one year, beginning at day 12 (`Fig~\ref{fig-taud-bsh}`{=latex}B); however, *tauD* expression in response to human decomposition was variable across taxonomic associations. Most *tauD* transcripts were associated with *Gammaproteobacteria*, *Actinobacteria*, *Betaproteobacteria*, *Alphaproteobacteria*, and fungi. While a majority of the *tauD* gene queries displayed reduced expression over time, expression of fungal-associated and a few *Betaproteobacteria*-associated *tauD* genes increased at day 58 (@sup-taud-tax). Sources of taurine in the human body include taurine absorbed from the diet and taurine produced from anaerobic microbial deconjugation of bile salts via bile salt hydrolase (BSH) enzymes [@urdaneta_interactions_2017]. Therefore, we also looked at expression of genes encoding BSH enzymes in decomposition soils. Expression of these genes was elevated at days 12, 58, and 86 before converging toward pre-decomposition levels at days 168 and 376 (`Fig~\ref{fig-taud-bsh}`{=latex}A). Hierarchical liner mixed effects (HLM) models showed that both *tauD* (HLM F = `r taud_f`, *p* = `r taud_p`) and BSH (F = `r bsh_f`, *p* `r bsh_p`) gene expression was significantly different over time (`Fig~\ref{fig-taud-bsh}`{=latex}A,B).

```{=tex}
% Place figure captions after the first paragraph in which they are cited.
\begin{figure}[!h]
\caption{{\bf Mean bile salt hydrolase, BSH, (A) and \textit{tauD}, taurine dioxygenase, (B) log2 normalized expression in controls (gold stars) and decomposition (boxplots) soils.}
Boxplots display the 25th and 75th quartiles and median log2 normalized values between all three individuals at each timepoint. ANOVA p-value is the result of a hierarchical linear mixed effects model accounting for repeated measures of each donor block, while letters denote the results of \textit{post-hoc} Tukey test.}
\label{fig-taud-bsh}
\end{figure}
```
![](../../figures/tauD_bsh_nlog2.png)

# Discussion

The goal of this study was to assess soil microbial gene expression in response to human decomposition. Metatranscriptomics were applied to soil samples collected over one year from below three decomposing human bodies. From this, we showed that microbial gene expression shifted over time, with samples reproducible between individuals. Additionally, we showed that gene expression profiles had not recovered to pre-decomposition conditions after one year. Comparison of control and decomposition expression profiles revealed that heat-shock proteins were elevated in response to decomposition. We also described expression patterns between decomposition timepoints, noting changes in functional gene categories at certain timepoints, in particular with respect to lipid, nitrogen and sulfur metabolism. 

## Gene expression patterns in decomposition soils

Gene expression profiles remained altered after one year of decomposition. It is unclear if soil microbial communities, in terms of gene expression profiles, have reached a new steady state as a result of decomposition, or if they would eventually return to pre-decomposition conditions. The parameters pH, EC, NH~4~^+^, NO~3~^-^, and total nitrogen (TN) exhibited differences (although not statistical) in these soils following a year of decomposition, reflected by changes in beta diversity which also did not show community recovery [@taylor_soil_2024]. This indicates that decomposition-derived materials present in the soil have not been fully cycled, and thus can continue to structure microbial communities and impact their function for extended periods of time. While nutrient pools and communities both demonstrate less rapid change at later time points in the study, there is not evidence suggesting an arrival at a steady-state post-disturbance microbial community. Further, Barton et al. (2020) [@barton_soil_2020] reported that soil phosphorus, nitrogen (total, ammonium, and nitrate), and electrical conductivity were still elevated above background levels after 500 days of human decomposition, suggesting decomposition events have long lasting effects on the local ecosystem. Together, this has implications for terrestrial ecosystem processing (*e.g.*, nutrient cycling, emission of greenhouse gasses, etc.), as we show that decomposition alters functional metabolism pathways within soil microbial communities. Further work with extended sample collections beyond one year are needed to address how long microbial gene expression is impacted.

Bacteria, fungi, and archaea were all represented in expressed genes throughout decomposition, suggesting that members of all three domains have the potential to contribute to decomposition processes and nutrient cycling. While a majority of annotated transcripts were identified as bacteria, fungal transcripts were the second most abundant group. Fungal transcripts made up almost half (seven of the top 15) of the significantly differentially expressed genes associated with decomposition-impacted soils. Additionally, with respect to expression shifts between decomposition timepoints, fungal transcripts were among the topmost upregulated genes at study day 86. The presence of fungal transcripts is not surprising as fungi are key decomposers, involved in the degradation of organic matter in terrestrial ecosystems [@van_der_wal_thready_2013]. It was interesting to see an increase in certain fungal transcripts, such as lipase, at study days 58 and 86 when soil oxygen began to recover. We would expect lipids to enter the soil as tissues are borken down during decomposition, however this did not inform the expression pattern of bacterial lipase genes, as they decreased during decomposition. This suggests that microbial activity in decomposition soils may be constrained by the changing chemical environment, potentially altered oxygen levels in the case of lipase gene expresssion. Similarly, prior work with these soils showed a relationship between fungal community composition and soil oxygen [@taylor_soil_2024], indicating that fungal communities underwent shifts in both community structure and activity in decomposition-impacted soils.

## Heat-shock/stress response

Soil microbial communities expressed stress response genes in response to human decomposition. Differential expression analysis identified increased expression of multiple heat shock proteins associated with the taxa *Xanthomonadales*, *Actinobacteria*, and fungi. Upon further investigation, expression of these genes increased through day 58 and remained high for the remainder of the year. Soil temperature was elevated relative to controls (up to 10&deg;C higher) between study days 8 and 80, while soil electrical conductivity increased up to 663 \textmu S\/cm (16X higher than background) through day 58 before slowly decreasing through the remainder of the study. Soil electrical conductivity (correlates with ionic strength [@essington_soil_2015] and can indicate soil salinity) has previously been shown to increase in decomposition soils [@keenan_mortality_2018; @fancher_evaluation_2017; @quaggiotto_dynamic_2019; @taylor_soil_2024]. As a result, we would expect these microbes to be experiencing both heat and osmotic stress during this period. Prior work has observed increased heat shock gene expression during salt stress in paddy soils [@peng_short-term_2017] and the presence of both heat and osmotic stress genes in desert soils along a salt gradient [@pandit_snapshot_2015], suggesting saline conditions can alter the expression of heat and/or osmotic stress genes. In our study we observed that stress response within soil microbial communities is stimulated during human decomposition, however, at this time, it is unclear if expression of these genes is in response to heat stress alone, or in combination with osmotic stress.

## Carbon-containing compounds

Human fat tissue contains lipids that are broken down during decomposition. Therefore, we assessed expression of triacylglycerol lipase genes in decomposition soils. Our results show that expression of triacylglycerol lipase genes was altered in response to decomposition, and these shifts differed between bacterial and fungal transcripts. Bacterial triacylglycerol lipase transcripts decreased in response to decomposition, while fungal triacylglycerol lipase transcripts increased. These results suggest that fungi may be important triacylglycerol degraders in human decomposition-impacted soils. Further, expression of these genes corresponded to changes in relative abundance of the fungal classes *Saccharomycetes*, *Sordariomycetes*, and *Eurotiomycetes* [@taylor_soil_2024]. These fungi have been previously associated with decomposition soils [@metcalf_microbial_2016; @fu_fungal_2019] and are known to contain triacylglycerol lipase genes in their genomes [@dujon_genome_2004; @haridas_genome_2013], suggesting that these organisms are important for lipid degradation in decomposition soils.

Our observation of an overall decrease in triacylglycerol lipase transcripts contrasts with previous work by Howard et al. (2010) [@howard_characterization_2010], who observed increased gene copy number of Group 1 lipase genes via qPCR during swine decomposition. Fatty acid composition differs in human compared to pig tissue [@notter_initial_2009], potentially altering the lipid profile available for microbes, leading to differences in decomposition products within the soil [@debruyn_comparative_2021]. These products can then directly or indirectly alter community composition and/or activity of functional proteins via substrate availability or the chemical environment. Further, decomposition of humans and pigs resulted in increased pH in soils below pigs, and decreased pH below humans [@debruyn_comparative_2021]. Altered pH and soil chemistry could result in a different functional potential and/or gene expression in decomposition-impacted soils, especially as it relates to lipase genes, as many triacylglycerol lipases have a pH optimum that is neutral to basic [@kok_characterization_1995; @hasan_influence_2006; @zouaoui_production_2012], so cells may be decreasing expression under acidic conditions in human decomposition soils. Availability of lipid species and changes to pH may select for taxa that favor these substrates/pH conditions; for example, Mason et al. (2022) [@mason_body_2022] suggested the abundance of the fungal taxa *Saccharomycetes* was related to antemortem BMI due to relative proportions of fat and muscle tissue.  

## Nitrogen enriched compounds/N-cycling

```{r}
#| include: false

do_avgs <- read_excel(here("data/SP_metadata.xlsx"), 
    sheet = "R") %>%
    mutate(Trt = as.factor(Trt),
           Study_day = as.factor(Study_day),
           Stage = factor(Stage, 
                          ordered = TRUE,
                          levels = c("Initial", "Active", "Advanced4","Advanced6","Advanced11","Skeletonization4"))) %>%
    pivot_longer(c(-Sample_name, -Date, -MetaT, -Trt, -Rep, -Study_con, -Study_day, -ADH_ambient, -ADH_internal, -ADH_soil, -Stage, -TBS, -timepoint), names_to = "parameter") %>%
    filter(parameter == "DO",
           Trt == "Decomposition") %>%
    summarize(avg = mean(value), .by = c("Study_day")) 
```

The human body is a concentrated source of nitrogen that is released into the surrounding soil during decomposition, therefore we also evaluated expression of genes involved in nitrogen cycling. Expression of common marker genes for nitrogen cycling was altered in decomposition soil and suggested nitrogen transformations during human decomposition are driven by soil oxygen concentrations with hydroxylamine as an important intermediate. We observed low or reduced expression of nitrification genes *nxrA* and *amoA* between days 12 and 86, during a period when oxygen was reduced to `r do_avgs %>% filter(Study_day == "12") %>% pull(avg) %>% round(., 0)`\% - `r do_avgs %>% filter(Study_day == "86") %>% pull(avg) %>% round(., 0)`\%. This was concomitant with accumulation of ammonium, which reached a maximum on day 12, and low nitrate conditions indicating that nitrification was inhibited. This period of reduced soil oxygen constraining nitrification was also described in a decomposition experiment with beaver carcasses Keenan et al. (2018) [@keenan_mortality_2018]. 

We observed increased expression of *hao*, which encodes the enzyme hydroxylamine dehydrogenase (HAO) at day 86 while oxygen was reduced (\~`r do_avgs %>% filter(Study_day == "86") %>% pull(avg) %>% round(., 0)`\%). This corresponded to simultaneous increases in expression of genes encoding nitic oxide reductase (*norB*) and nitrous oxide reductase (*nosZ*). Traditionally HAO has been thought to process hydroxylamine to nitrite during nitrification, while NorB and NosZ are enzymes involved in the last two steps of denitrification converting nitric oxide (NO) to dinitrogen gas (N2). However, recent work has suggested hydroxylamine can be converted to nitric oxide (NO), as well as can interact with multiple phases of the nitrogen cycle [@soler-jofra_hydroxylamine_2021]. Even though *amoA* expression was shown to decrease during reduced oxygen conditions, *amoA* transcripts were still present and likely able to convert ammonium to hydroxylamine as soil oxygen was not completely depleted during decomposition. Additionally, a previous study reported that the growth of the ammonia oxidizing bacteria *Nitrosomonas europaea* under anoxic conditions lead to accumulation of hydroxylamine in a chemostat bioreactor [@yu_nitrosomonas_2018], suggesting anaerobic ammonium oxidation (anammox) may also be occurring in decomposition soils. However, we did not observe increases in *nirK* expression, which might suggest conversion of nitrite to NO for use in the anammox pathway. NO produced via HAO activity may be used for anammox in these soils; however, the role of hydroxylamine as an intermediate in anammox is still debated [@soler-jofra_hydroxylamine_2021]. Therefore, our current hypothesis is that hydroxylamine accumulates under anaerobic conditions during decomposition, which can then be converted to NO by HAO. This NO would then be present for anaerobic denitrifying bacteria to convert to nitrous oxide (N2O) by NorB and finally to N2 by NosZ. Keenan et al. (2018) [@keenan_mortality_2018] also noted a brief increase in N2O emissions, which suggests denitrification was occurring during this phase of reduced soil oxygen concentrations. 

As soils fully reoxygenated by day 168, we observed increased expression of genes encoding enzymes involved in aerobic nitrification, *amoA* and *nxrR*. Nitrification is an oxygen-dependent process which would be converting the accumulated ammonium to nitrate; the increase in nitrate concentrations may then serve as a substrate for denitrification. We observed increased expression of marker genes encoding all four enzymes in the complete dissimilatory denitrification pathway (*narG*, *nirK*, *norB*, and *nosZ*) at day 376. Increased expression of nitrification and denitrification marker genes is consistent with accumulation of nitrite, nitrate, and N2O after oxygen is reintroduced to soils described in Keenan et al. (2018) [@keenan_mortality_2018]. Together, gene expression patterns in our study provide further insight into nitrogen transformations in during vertebrate decomposition, suggesting an important role of hydroxylamine. 

## Sulfur-containing compounds

Sulfur is present in various organic molecules, including taurine, a sulfur- and nitrogen-containing acid involved in bile acid formation [@urdaneta_interactions_2017]. Taurine is present in the human body, where it can be absorbed from the diet or synthesized in the liver [@seidel_taurine_2019]. However, taurine is also produced as a byproduct of the deconjugation of bile salts via bile salt hydrolases (BSH) present in the anaerobic gut taxa *Lactobacillus* and *Clostridium* [@urdaneta_interactions_2017]. In our study, we observed increased expression of genes encoding BSH enzymes between days 12 and 86. Given that increased expression of BSH genes corresponded to the beginning of active decomposition, when decomposition products were observed to enter the soil, and the period of reduced dissolved oxygen in our study, it is likely that taurine accumulation is the result of BSH enzyme activity by anaerobic microorganisms. While we did not measure taurine concentrations in this study, our results correspond to previous decomposition studies that report accumulation of taurine in various organs and body regions [@mora-ortiz_thanatometabolomics_2019; @locci_1h_2019; @zelentsova_post-mortem_2016] and soils [@debruyn_comparative_2021; @hoeland_katharina_investigating_2021] during decomposition via metabolomics, and increased relative abundance of *Clostridium *and *Lactobacillus* within the body [@javan_human_2016; @javan_cadaver_2017; @debruyn_postmortem_2017] and in decomposition soils [@cobaugh_functional_2015] via DNA sequencing methods, including in these soils [@taylor_soil_2024].
 
One pathway of taurine metabolism is through desulfurization via the \textalpha-ketoglutarate-dependent enzyme taurine dioxygenase (TauD). Specifically, this enzyme, encoded by the gene *tauD*, converts 2-oxoglutarate and taurine to produce aminoacetaldehyde, succinate, sulfite, and CO~2~ [@cook_metabolism_2006]. Succinate and sulfite from this reaction can then be used for the citric acid cycle and sulfur metabolism, respectively. Given increased BSH expression in our study and reported taurine accumulation in others, we would expect taurine to be present for microbial metabolism by TauD. However, we observed a general decrease in *tauD* expression between days 12 through 376. This trend was driven by reduced expression of *tauD* transcripts associated with *Proteobacteria*, *Gammaproteobacteria*, and *Actinobacteria* whose relative abundance have been shown to remain consistent or increase during human decomposition [@cobaugh_functional_2015], suggesting that *tauD* expression is downregulated under decomposition conditions. However, we noted that expression of *tauD* genes associated with fungi and a few *Betaproteobacteria* displayed increased expression at day 58, corresponding to increased expression of bile salt hydrolases (BSH) between days 12 and 86. The reduction in *tauD* expression may be due to sulfur availability. We did not measure sulfur species in this experiment; however, others have observed increased sulfur concentrations in decomposition-impacted soils [@aitkenhead-peterson_mapping_2012; @taylor_soil_2023; @vass_time_1992]. Thus, sulfur scavenging pathways such as taurine desulfurization by TauD [@kertesz_riding_2000], whose genes are expressed under sulfur-limiting conditions, likely display reduced expression under sulfur replete conditions. Additionally, taurine may be processed through other pathways. For example, taurine can be deaminated by taurine dehydrogenase to produce sulfite and acetyl-CoA for carbon metabolism [@cook_metabolism_2006; @bruggemann_enzymes_2004]. Overall, our results suggest that human decomposition has potential impacts on soil sulfur biogeochemistry through deposition of inorganic (sulfate) and organic (sulfur-containing amino acids) sulfur compounds. 
 
# Conclusion

This study represents the first investigation of soil microbial gene expression during human decomposition. Metatranscriptomic analysis of soils from three human individuals over one year shows that decomposition impacted microbial community gene expression profiles, exhibiting functional shifts over time. This included altered expression of genes involved in lipid, N and S metabolism as microbes processed the nutrient-rich tissues of the human body. Additionally, we noted that functionality within decomposition-impacted soils was still affected after one year and had not returned to starting or background conditions. Together, these results show that vertebrate decomposition has lasting impacts on local soil ecosystems, including soil microbial communities. These results have important implications for understanding biogeochemical changes due to vertebrate mortality events in terrestrial ecosystems.

# Materials and Methods

## Study design 

In February 2018, three deceased male human subjects (hereafter, donors) were placed supine on the soil surface at the University of Tennessee Anthropology Research Facility (ARF) and allowed to decompose. Located in Knoxville, TN (35 56' 28" N, 83 56' 25" W) the ARF is a roughly 2-acre outdoor facility dedicated to studying human decomposition [@keenan_spatial_2018]. The soils at the ARF are comprised of the Loyston-Talbott-Rock outcrop (LtD) and Coghill-Corryton (CcD) complexes. LtD soils are a silty clay loam and channery clay overlaying lithic bedrock, while CcD soils are comprised of clay from weathered quartz limestone [@keenan_spatial_2018; @taylor_soil_2024]. A site that had not been previously exposed to decomposition was used for this study. 

The decomposition field experiment is fully described in Taylor et al. (2024) [@taylor_soil_2024]. Briefly, experiments were conducted in a block design, where each block consisted of one decomposition site and one control site [@taylor_soil_2024]. In total three blocks, *i.e.*, three donors paired with three respective control sites, were included in the study. Each control site was chosen in a manner to ensure their location was uphill and roughly 2 m away from decomposition sites [@taylor_soil_2024]. Donor internal temperatures were recorded by probes located in the abdomen, while ambient air temperatures were monitored via sensors located roughly 50 cm above the soil surface. Soil temperature and salinity were measured with sensors placed directly underneath each individual (Decagon Devices, GS3) [@taylor_soil_2024]. Donor ages ranged from 65 to 86 and were within 1 kg of each other with regard to weight (90.7 to 91.6 kg); donor BMI varied between 27.7 to 29.6 [@taylor_soil_2024]. 

## Sampling and physiochemistry

Decomposition of all subjects was observed for one year. During the one-year study period, soils were sampled at 20 timepoints chosen to correspond with morphological stages of decomposition as described by [@payne_summer_1965]. Once advanced decay was reached, soils were collected at intervals of 350 accumulated degree days (ADD), calculated using ambient air temperatures, up to one year. All soil cores were taken using a 1.9 cm (3/4 inch) diameter soil auger to a depth of 16 cm. Soils were divided into two depth fractions: 0-1 cm (interface) and 1-16 cm (core) for the analyses reported in Taylor et al. (2024) [@taylor_soil_2024]; the entire 0 to 16 cm core was used for this current study. Decomposition soils were taken from directly beneath the cadavers, taking care to not re-sample the same location more than once. At the time of sampling, soil dissolved oxygen was measured in triplicate using an Orion Star  A329 pH/ISE/Conductivity/Dissolved Oxygen portable multiparameter meter (ThermoFisher) [@taylor_soil_2024].

A subset of 6 study timepoints were chosen for metatranscriptomics analysis. Study days 0, 12, 58, 86, 168, and 376 were chosen as they represented disctinct morphological and soil biogeochemical stages during decomposition. Study day 0 was chosen as a baseline sample prior to cadaver placement. Study day 12 was chosen as this was they start of active decomposition and corresponded to maximum soil ammonium concentrations and minimum soil oxygen (approximately 39%). Study day 58 was chosen as this sample represented the pH minimum, and respirationn and soil temperature were at a maximum [@taylor_soil_2024]. Additionally, ammonium concentrations began to decrease around day 58. Study day 86 was chosen as the time period when soil oxygen started to recover and nitrate levels began to increase. Study day 168 was chosen as nitrate was at its maximum and soil dissolved oxygen had returned to 99%. Finally, day 376 was chosen to represent the end of the study, approximately 1 year since cadaver palcement. Each study day was represented by four soil samples for RNA extraxtion: one pooled control sample which was a mix of the three control locations, plus one sample from each of the three donors, yielding a total of 24 samples for this study.

Soil samples were transported back to the lab at the University of Tennessee (Knoxville, TN) and processed within 24 hours of collection. Soils were homogenized by hand to remove insect larvae, roots, rocks, and other debris (> 2 mm). A subset of soils were used to measure pH, electrical conductivity (EC), and evolved CO~2~ as described in Taylor (2020). Soil nitrogen species (NH~4~^+^, NO~3~^-^) and total carbon (TC) and nitrogen (TN) were measured in all soil samples as described in [@taylor_soil_2024]. Reported values for soil physiochemistry represent the full 16 cm core; estimated by summing interface and core values reported by Taylor et. al, (2024) [@taylor_soil_2024] in 1:16 and 15:16 ratios, respectively. Control reported here are means of the three experimental controls that were unimpacted by decomposition.

Roughly 10 g of soil was reserved for nucleic acid extraction, placed in a 4 oz. Whirl-Pak\texttrademark\ bag (Nasco), and flash frozen in liquid nitrogen. All samples were stored at -80&deg;C until further analysis. Bacterial and fungal community composition was assessed via amplicon sequencing of the 16S rRNA gene and ITS2 region as described in Taylor et al. (2024). 


## RNA Extraction and Sequencing

RNA was extracted from 2 g of soil using Qiagen's RNeasy\textregistered\ PowerSoil\textregistered\ Total RNA kit (catalog no. 12866-25). Manufacturers instructions were followed with a few modifications. Soils became saline during decomposition; therefore, we followed manufacturers suggestion and incubated all extracts at -20&deg;C following addition of solution SR4 (step 9) to decrease salt precipitation. All RNA samples were resuspended in 40 \textmu l of Solution SR7. RNA concentrations were assessed fluorometrically using the Qubit\textregistered\ RNA HS assay (catalog no. Q32852) with 1 \textmu l of RNA. DNA contamination was removed by DNase treating RNA extracts twice using Qiagens DNase Max\textregistered\ kit in 50 \textmu l reactions. RNA concentrations were remeasured after DNase treatment. PCR with general V4 16S rRNA gene primers [@apprill_minor_2015; @parada_every_2016] was conducted using RNA extracts as the template to confirm removal of all DNA prior to sequencing. RNA aliquots were shipped to HudsonAlpha Discovery (Huntsville, AL) for library preparation and RNA sequencing. Dual-indexed libraries were prepared using the Illumina Stranded Total RNA prep with ribosomal RNA depletion via ligation with Ribo-Zero Plus. Libraries were then pooled and sequenced on Illuminas NovaSeq 6000 v4 platform, resulting in demultiplexed fastq files for each sample. 

## Bioinformatics

Read quality control (QC) was conducted in KBase [@arkin_kbase_2018] using Trimmomatic [@bolger_trimmomatic_2014]. Paired fastq files were imported to KBase through Globus. Poor quality reads were removed, and adapters trimmed via Trimmomatic (v0.36) using default settings and the TruSeq3-PE-2 adapter file. After QC check with FastQC, trimmed libraries were exported as fastq files from KBase through Globus. Remaining ribosomal RNA was filtered using bbmap (maxindel = 20, minid = 0.93) from the Joint Genome Institutes (JGI) bbtools suite [@bushnell_bbmap_nodate]. After this step, all non-ribosomal reads from all 24 samples were merged into one file. This file was then used to co-assemble reads into contigs using the de novo assembler MEGAHIT (v1.2.9) [@li_megahit_2015] (--12 --k-min 23, --k-max 123, --k-step 10). 

Gene identification and annotation from co-assembled contigs was performed using Prodigal [@hyatt_prodigal_2010] and eggNOG mapper [@cantalapiedra_eggnog-mapper_2021], respectively. Briefly, the fastq containing all contigs was summitted to Prodigal (v2.6.3) for protein coding gene predication for a meta-sample (-p meta -f gff). Next, predicated genes were functionally and taxonomically annotated using eggNOG mapper (v2.1.6) using basic settings to perform a diamond blastp search [@buchfink_sensitive_2021]. Only genes that were both functionally and taxonomically annotated by one of the databases used by eggNOG mapper and identified as bacterial, archaeal, or fungal were chosen as genes of interest. Transcript counts for all genes of interest were obtained by mapping reads from each respective sample to genes of interest obtained from co-assembly using QIAGEN CLC Genomics Workbench 20.0 (https://digitalinsights.qiagen.com/).

## Differential Expression

Transcript counts from all samples were combined in a single workable data file and imported into R for differential expression analysis using the R packages edgeR [@robinson_edger_2010] and limma [@smyth_limma_2005] following a modified pipeline by Phipson et al. (2020) [@phipson_differential_2020]. Briefly, the transcript count table was imported into R and converted to a DGElist object. Genes without sufficient counts for statistical analysis were removed to increase power using the edgeR function filterByExpr(). Raw counts were then log2 normalized and gene expression profiles compared via multidimensional scaling (MDS) and hierarchical clustering. For differential expression analysis, raw filtered reads were normalized using edgeRs trimmed mean of M values (TMM) normalization using the function calcNormFactors(). TMM normalized reads were then log2 transformed using limmas voom() and differential expression assessed. Empirical Bayes shrinkage was used correct to p-values for false discovery rates. The topmost up and down regulated genes for each comparison, determined by log2 fold change and adjusted p-values, were then reported. Expression of certain genes were assessed after performing transcripts per million (TPM) normalization and statistical analyses with a combination of analysis of variance (ANOVA) and post-hoc Tukey tests. ANOVA across all timepoints were applied to hierarchical linear mixed effects models to account for repeated sampling within each donor block.


# Availability of data and materials

RNA sequence files from the Novoseq intrument can be found at XXXX. The datasets supporting the conclusions of this article are available at the GitHub repository [Mason_MetaT_XXX_2024](https://github.com/amason30/Mason_MetaT_XXX_2024)."

# References
<!-- the pattern below controls the placement of the references -->
:::{#refs}
:::


# Acknowledgements

We would like to thank the Forensic Anthropology Center at the University of Tennessee-Knoxville for their help in setting up field experiments. We would like to thank Mary Davis for her help in managing the field site and helping to obtain donors for this work. This research was funded by a National Institute of Justice Award (DOJ-NIJ-2017-R2-CX-0008) to LST and JMD.

# Supplementary Information {.unnumbered}

::: {#sup-chem-plot}
![](../../figures/soilchem_summary.png)

Figure S1. Soil physiochemical parameters in decomposition soils during the one-year study. Data is shown for each individual donor: SP1 (gold), SP2 (blue), and SP2 (black). Values for the full 16 cm core samples were estimated by summing values interface (0-1 cm) and core (0-16 cm) reported by Taylor et al, (2024) in 1:16 and 15:16 ratios, respectively. Controls reported here are means of three experimental controls that were unimpacted by decomposition and are represented by stars.
:::

::: {#sup-heatmap-plot}
![](../../figures/Fig_S2.png)

Figure S2. Hierarchical clustering heatmap showing the log counts per million (CPM) of the top 500 most variable genes across samples. Variable genes were determined by selecting genes with the highest variance in gene expression. Samples are clustered along the x-axis using Euclidean distances between samples and colored by study day.
:::

::: {#sup-tbl-cca-panova}
Table S1. Permutational analysis of variance (PERMANOVA) results identifying significant environmental parameters which explain some of the variation in soil gene expression profiles. Environmental parameter data is from Taylor et al. (2024). Variables with p < 0.05 are indicated in bold.
:::

::: {#sup-t30-trt-plot}
![](../../figures/Trt_DE_top20_barplot.png)

Figure S3. Top 40 up- and down-regulated genes in controls relative to decomposition soils across all study days, colored by COG functional category (A) and taxonomic annotation (B). Positive values denote higher expression in controls, while negative values are higher in decomposition soils.
:::

::: {#sup-t30-trt-tbl}

Table S2. Top 20 most up- and down-regulated gene queries, determined by log2 fold change and adjusted p-values, in control relative to decomposition soils. Positive log2 fold change values represent genes whose expression was higher in control soils, while negative log2 fold change values were higher in decomposition soils. Taxonomic annotation, COG categories, gene description, gene names, and EC were assigned via eggNOG-mapper.
:::


::: {#sup-t20-day-tbl}
Table S3. Top 10 most up- and down-regulated genes, determined by log2 fold change and adjusted p-values, for each sequential timepoint comparison. Positive log2 fold change values represent genes whose expression was higher in the later decomposition timepoint soils, while negative log2 fold change values are higher in earlier decomposition timepoint soils. Taxonomic annotation, COG categories, gene names, and EC were assigned via eggNOG-mapper. The comparison column distinguishes each timepoint comparison.
:::


::: {#sup-taud-tax}
![](../../figures/tauD_nlog2_taxonomy.png)

Figure S4. Mean normalized log2 expression of *tauD* genes by taxonomic association (color) in control and decomposition soils at each study day. Each line represents one *tauD* gene query, while color denotes taxonomic association as determined by eggNOG-mapper.
:::


